client/orm/types.go

```diff
@@ -405,6 +405,15 @@ type QuerySeter interface {
 	// 	Found int
 	// }
 	RowsToStruct(ptrStruct interface{}, keyCol, valueCol string) (int64, error)
+	// aggregate func.
+	// for example:
+	// type result struct {
+	//  DeptName string
+	//	Total    int
+	// }
+	// var res []result
+	//  o.QueryTable("dept_info").Aggregate("dept_name,sum(salary) as total").GroupBy("dept_name").All(&res)
+	Aggregate(s string) QuerySeter
```

Comment:
From this method, my understanding is that we could not use `qs.Count(XXX).GroupBy('xxx').All`?

Reply:
This method returns two values: the total number of queries and error information.
It is different from the aggregate function `count`.
https://github.com/astaxie/beego/blob/26bc9e72645344d65dc79fd39f2ec0dddeb3d617/client/orm/types.go#L326-L329